pom.xml

```diff
@@ -1132,7 +1132,7 @@
                             <goals>
                                 <goal>enforce</goal>
                             </goals>
-                            <phase>compile</phase>
+                            <phase>none</phase>
```

Comment:
This turns off enforcement for the entire build. You need to add this plugin config to testsuite/integration/microprofile-tck/certification/pom.xml and set the phase to 'none' there.

Reply:
Thanks! for the review, fixed it.